### ISSUE TITLE
Publish all from folder does not work

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#17140]
From the browser, when you right-click on a folder and hit `publish all`, nothing happens.  Instead, you get this in your logs:

```
10:38:21.192  WARN  action.ActionMapping - Unable to find '/common/forward_js.jsp' forward.
```
 
![publish-all-broke](https://user-images.githubusercontent.com/934364/64265238-8c29a580-cf00-11e9-8544-7b3970f20327.gif)

**Comment by nollymar** [comment]
PR: https://github.com/dotCMS/core/pull/17199

**Comment by erickgonzalez** [comment]
Working as it should. Shows a page with all the content that is unpublish and ask for confirmation to publish it.

**Comment by bryanboza** [comment]
We need some work here: 
- The confirmation window needs some styling help (table)
- We are not showing the corresponding icons SC: https://screencast.com/t/wtxiL8cs
- If on the confirmation window you click the cancel button, we need to redirect again to the site browser, right now we are just leaving a blank page SC: https://screencast.com/t/aBHDAPYjfb
- If you have a page and this contains a combination of publish and unpublished content, we are showing an error in the process and we are unable to publish the page 
ST: https://gist.github.com/bryanboza/53f1a8a0a1c2167d95f3c4f6ad3f7f56
SC:https://screencast.com/t/Rvo8pBNm9kCT

**Comment by nollymar** [comment]
@wezell @bryanboza I think it is convenient to move @bryanboza 's observations to a new card because they are not part of the scope of this issue

**Comment by wezell** [comment]
@nollymar I agree a new.card would be best

**Comment by nollymar** [comment]
A new card with the suggested improvements has been created here: https://github.com/dotCMS/core/issues/17287

**Comment by bryanboza** [comment]
Agree with you @wezell and @nollymar, but in order to pass QA this card, we need to fix the point 4 since I'm unable to reproduce on demo, and you are able to reproduce exactly with the same steps of the original issue.

**Comment by wezell** [comment]
@bryanboza is correct ,we need to push all content through the Default "Publish" workflow.

**Comment by bryanboza** [comment]
In this process, if you manually unpublish some page//asset from the folder manually, and then try to publish(all) to the entire folder, we are not publishing those assets in the first time, you need to do this twice.

**Comment by nollymar** [comment]
I was not able to reproduce the point 4 (If you have a page and this contains a combination of publish and unpublished content, we are showing an error in the process and we are unable to publish the page). However, fixed point 3 (If on the confirmation window you click the cancel button, we need to redirect again to the site browser, right now we are just leaving a blank page) in here: https://github.com/dotCMS/core/pull/17308

**Comment by nollymar** [comment]
Fixed issue with the Unpublish operation reported by @bryanboza https://github.com/dotCMS/core/issues/17140#issuecomment-533657748

PR: https://github.com/dotCMS/core/pull/17359

I was not able to reproduce Point 4 reported here https://github.com/dotCMS/core/issues/17140#issuecomment-532318988

**Comment by craigWagner99** [comment]
Another wrinkle in the Publish All from Folder functionality: 

When using the "Publish (All)" action on a folder in 5.x, the list of contents to publish is not displayed correctly. The "Title" field always shows the value "**Contentlet Title**" instead of the actual title.

Steps to recreate:
- Unpublish a page that contains content, or edit + Save (but not Publish)
- Right click on the parent folder, choose "Publish (all)"
- On the next screen, all of the contents on the pages to be published will have the title "Contentlet Title"

![image](https://user-images.githubusercontent.com/49727631/65707504-4c169800-e052-11e9-9e80-2501a2a79b02.png)

**Comment by dsilvam** [comment]
PR 2: https://github.com/dotCMS/core/pull/17375

**Comment by dsilvam** [comment]
Passed Internal QA: Publishing the folder effectively publishes all the unpublished assets in it *and* now are showing better information about the unpublished assets to publish in the preview screen.

**Comment by bryanboza** [comment]
Fixed, new improvement reported here #17387
